src/main/scala/subsystem/HasTiles.scala

```diff
@@ -348,8 +375,8 @@ trait HasTiles extends InstantiatesTiles with HasCoreMonitorBundles with Default
   implicit val p: Parameters
 
   // connect all the tiles to interconnect attachment points made available in this subsystem context
-  tileAttachParams.zip(tiles).foreach { case (params, t) =>
-    params.connect(t.asInstanceOf[params.TileType], this.asInstanceOf[params.TileContextType])
+  tileAttachParams.zip(tile_prci_domains).foreach { case (params, td) =>
+    params.connect(td.asInstanceOf[TilePRCIDomain[params.TileType]], this.asInstanceOf[params.TileContextType])
```

Comment:
it seems desirable that someday we might want to support more than 1 tile per PRCI domain wrapper

Reply:
I certainly wouldn't rule that out, but if those tiles share some resources like e.g. a shared cache probably all three of them live inside a ClusteredTilesWrapper, which might either have its own clock sink node or just be wrapped in one of these clock domain wrappers itself. Alternatively we can always (configurably?) make wrappers around these wrappers... put another way, maybe this wrapper is just a `DevicesSubsystem` with a single tile "device" inside it, if you upstreamed that and we re-expressed it that way :)